src/communications/TravelBroker.java

```diff
@@ -46,35 +50,101 @@ public void start(int backlog){
                 });
                 travelThread.start();
             }
-        }catch(Exception e){
+        } catch (Exception e) {
             e.printStackTrace();
         }
     }
-    public void handleRequest(String message, Socket socket){
-        //MessageSplit [0] = WhatAmI, [1] = ProcessId, [2] = Message
+
+    public void handleRequest(String message, Socket socket) {
+        //MessageSplit [0] = WhatAmI, [1] = processId, [2] = Message
         String[] messageSplit = message.split(" ", 3);
-        UUID ProcessId = UUID.fromString(messageSplit[1]);
+        UUID processId = UUID.fromString(messageSplit[1]);
         String whatAmI = messageSplit[0];
         switch (whatAmI) {
             case "ClientRq":
                 List<BookingRequest> bookingRequests = BookingRequestParser.parse(messageSplit[2]);
                 for (BookingRequest bookingRequest : bookingRequests) {
                     //Send Flight BookingRq to Messagebroker
-                    String newMessage = "BookingRq " + ProcessId + " " + bookingRequest.type() + " " + bookingRequest.name() + " " + bookingRequest.quantity();
+                    String newMessage = "BookingRq " + processId + " " + bookingRequest.type() + " " + bookingRequest.name() + " " + bookingRequest.quantity();
                     MessageSenderService.sendMessageToMessageBroker(newMessage);
                 }
                 break;
             case "Confirmation":
-                //<WhatAmI> <ProcessId> < --flight '12312' true --hotel 'Hotel' false>
+                handleConfirmation(message, processId);
                 break;
 
             case "CancellationConfirmation":
+                String clientMessage = "ClientConfirmation " + processId + " " + "false";
+                MessageSenderService.sendMessageToMessageBroker(clientMessage);
                 break;
 
             default:
-
+                System.out.println("TravelBroker - Message not recognized: " + message);
                 break;
         }
     }
+
+    public void handleConfirmation(String message, UUID processId){
```

Comment:
What the fuck

Reply:
Das muss doch safe iwi besser gehen. Das sieht echt kacke aus. Ich weiß nicht ob er das im Code review so durchgehen lässt. Hast du überlegt wegen switch case?

Reply:
schön isses nicht, aber switch case würd nur semi besser passen weil ich da auch mehrere switches bräuchte

Reply:
Ich überleg mir was, geht bestimmt schöner wenn ich teile davon auslager